application/cms/models.py

```diff
@@ -257,7 +257,9 @@ def __lt__(self, other):
 
     slug = db.Column(db.String(255))  # TODO: Remove
     review_token = db.Column(db.String())  # used for review page URLs
-    description = db.Column(db.Text)  # TOPIC PAGES ONLY: a sentence below topic heading on homepage
+    description = db.Column(
+        db.Text
+    )  # TOPIC PAGES: a sentence below topic heading on homepage.  MEASURE PAGES = a short summary used by search engines and social sharing.
```

Comment:
Feels reasonable to drop the line about 'TOPIC PAGES' given that we no longer read/write that data.

Reply:
Yeah definitely drop the TOIPIC PAGES bit of this comment. They will be all gone soon.

Reply:
👍